libs/blocks/global-navigation/global-navigation.js

```diff
@@ -280,6 +280,10 @@ class Gnav {
     if (this.useUniversalNav) {
       delete this.blocks.profile;
       this.blocks.universalNav = toFragment`<div class="feds-utilities"></div>`;
+      this.blocks.universalNav.addEventListener('click', () => {
+        const isExpanded = this.isToggleExpanded();
+        if (isExpanded) this.toggleMenuMobile();
```

Comment:
This could be just one line, since `isExpanded` is not used anywhere else, `if (this.isToggleExpanded()) this.toggleMenuMobile();`